protobuf/sliverpb/constants.go

```diff
@@ -279,6 +279,11 @@ const (
 	MsgMvReq
 	// MsgMv - Confirms the success/failure of the mv request (resp to MsgMvReq)
 	MsgMv
+
+	// MsgCurrentTokenOwnerReq - Request to query the thread token owner
+	MsgCurrentTokenOwnerReq
+	// MsgCurrentTokenOwner - Replies with the current thread owner (resp to MsfCurrentToken)
+	MsgCurrentTokenOwner
```

Comment:
Strange behavior here. If I set these constants and generate an implant, the error message I get back is `unknown message type` (server error) and a client error of  `unknown envelope`

If I come back to the constants and swap the order, but use the old implant, now everything works. 

It sounds like I mixed up an order of `CurrentTokenOwner{Req}` somewhere else maybe?

Reply:
resolved: it's always a spelling mistake